src/Mod/PartDesign/Gui/Utils.h

```diff
@@ -23,7 +23,7 @@
 #ifndef UTILS_H_CS5LK2ZQ
 #define UTILS_H_CS5LK2ZQ
 
-/** \file Utils.h
+/** \file PartDesign/Gui/Utils.h
  *  This fiel contains some utility function used over PartDesignGui module
```

Comment:
It's a reference to a file i.e. it should be \file but doxygen complains about ambiguity. There are two files in its realm that are called Utils.h and it wants to know specifically which one is referenced.